projects/kit/components/combo-box/combo-box.component.ts

```diff
@@ -183,9 +183,8 @@ export class TuiComboBoxComponent<T>
 
         const match = this.accessor?.getOptions().find(item => this.isStrictMatch(item));
 
-        if (match !== undefined) {
+        if (match) {
```

Comment:
Value can be falsy, previous check was correct

Reply:
revert